tests/testutils/pdata_to_resource_metrics.go

```diff
@@ -0,0 +1,195 @@
+package testutils
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	tracetranslator "go.opentelemetry.io/collector/translator/trace"
+)
+
+// Returns a ResourceMetrics item generated from pdata.Metrics content.  At this time histograms and summaries
+// aren't supported.
+func PDataToResourceMetrics(pdataMetrics ...pdata.Metrics) (ResourceMetrics, error) {
```

Comment:
Very large function with a lot of indentation. Difficult to read, consider splitting.

Reply:
Broke out some metric adding functions.  This ok?